src/client/components/features/OffreDeStage/Déposer/Étape3Localisation/StageDeposerOffreFormulaireÉtape3Localisation.tsx

```diff
@@ -50,6 +50,11 @@ export default function StageDeposerOffreFormulaireÉtape3Localisation() {
 
 	const localStorageLocalisation = useLocalStorage<OffreDeStageDeposee.Localisation>(ETAPE_LOCALISATION);
 	const informationsLocalisation = localStorageLocalisation.get();
+	const paysDefaultValueLabel = paysList.find((pays) => pays.code === informationsLocalisation?.pays)?.libellé;
```

Comment:
```suggestion
  function getPaysDefaultValue(): PaysOption | undefined {
    const storedPaysCode =  informationsLocalisation?.pays
    if(!storedPaysCode) return undefined;
    const paysLabel = paysList.find((pays) => pays.code === storedPaysCode)?.libellé
      if(!paysLabel) {
        return undefined
      }
    return {
      code: storedPaysCode,
		  label: paysLabel,
    }
  }
```
Mieux vaut grouper ce qui doivent aller ensemble dans une fonction

Reply:
Pas de test en plus par rapport à cette logique supplémentaire ?